frontend/src/styles.css

```diff
@@ -68,21 +63,11 @@
         border-radius: 4px;
         box-sizing: border-box;
       }
-      .field button {
```

Comment:
You can just use the button style options here: 
https://primeng.org/button#basic

Just add the appropriate [thing]="true" options

Reply:
Example, for rounded buttons do [rounded]="true"

https://primeng.org/button#rounded

Reply:
would it be easier to assign a general style to the buttons if wed like the style to apply across all buttons?

Reply:
I don't really see us as wanting to / needing to make a custom button style / look that is more complex than what PrimeNG already provides.

Reply:
We can just add the class to all the buttons that need them. Then we get configurability and we aren't going to break any primeNG things that rely on buttons being styled a certain way (for instance some PrimeNG components use button, so if we change button we might break those things)

Reply:
This is why we typically avoid using the styles.css file